src/MagnumPlugins/ObjImporter/Test/multiMaterial.mtl

```diff
@@ -0,0 +1,8 @@
+newmtl mat_0
+Ka 1 1 1
+map_Ka Textures/mat_0.tga
+
+
+newmtl mat_1
+Ka 1 1 1
+map_Ka Textures/mat_1.tga
```

Comment:
But then we never test files missing newlines at end?
Just saying, I think I found a bug in `parse()`, because another file was missing that. I can extract that into separate files, though, which are explicitly named `missingNewlineAtEof.obj/.mtl`